crates/uv-workspace/src/workspace.rs

```diff
@@ -26,7 +26,17 @@ impl Workspace {
         debug!("Loading user configuration from: `{}`", file.display());
         match read_file(&file) {
             Ok(options) => Ok(Some(Self { options, root })),
-            Err(WorkspaceError::Io(err)) if err.kind() == std::io::ErrorKind::NotFound => Ok(None),
+            Err(WorkspaceError::Io(err)) if matches!(err.kind(), std::io::ErrorKind::NotFound) => {
```

Comment:
Why this change? Just curious.

Reply:
Oops, mistake. I started by changing it to match against NotADirectory.